src/app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.h

```diff
@@ -61,6 +61,14 @@ class SmokeCoAlarmServer
      */
     bool SetExpressedState(chip::EndpointId endpointId, ExpressedStateEnum newExpressedState);
 
+    /**
+     * @brief Set the highest level of Expressed State according to priorityOrder
```

Comment:
What does the `Auto` in the name mean?

Comment-wise the endpointId being the endpointid is obvious.

The users of this method seems to never care about true/false result. Please upate the code to use true/false or never return it (and just log on error).

Reply:
It's changed to `void SetExpressedStateByPriority`.